applications/difx2fits/src/sniffer.c

```diff
@@ -1183,53 +1148,9 @@ int feedSnifferFITS(Sniffer *S, const DifxVis *dv)
 
 	if(configId != S->configId)
 	{
-		int i;
-		int bbc;
-		int writeNewConfig;
-
-		if(S->configId == 0 || S->D->config[configId].freqSetId != S->D->config[S->configId].freqSetId)
-		{
-			writeNewConfig = 1;
-		}
-		else
-		{
-			writeNewConfig = 0;
-		}
-
 		S->nIF = dfs->nIF;
 		S->nPol = dc->nPol;
 		S->configId = configId;
-		++S->nConfigsUsed;
```

Comment:
Similarly here I thought most of this code actually was present, and it was only from line 1220 that was actually the edited/added code. I'm looking at commit 11039 from SVN (Walter, 15 Aug 2023).